packages/frontend/src/components/nft/NFTDetail.js

```diff
@@ -0,0 +1,196 @@
+
+const UserIcon = styled.div`
+    background-size: 21px;
+    flex: 0 0 40px;
+    height: 40px;
+    border-radius: 50%;
+    background-color: #f8f8f8;
+    text-align: center;
+    margin: 0 12px 0 0;
+    
+    svg {
+        width: 26px;
+        height: 26px;
+        margin: 7px;
+    }
+
+    @media (min-width: 940px) {
+        display: inline-block;
+    }
+`;
+
+function transferSVG() {
+    return (
+        <svg className="transfer-svg" width="22" height="22" viewBox="0 0 22 22" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path d="M21 1L14 21L10 12L1 8L21 1Z" stroke="#A2A2A8" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"/>
+          <path d="M21 1L10 12" stroke="#A2A2A8" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"/>
+        </svg>
+    );
+}
+
+export function NFTDetail({ match, location, history }) {
+    const accountId = useSelector(selectAccountId);
+    const fungibleTokens = useFungibleTokensIncludingNEAR();
+    const nearBalance = fungibleTokens[0].balance;
+
+    const contractId = match.params.contractId;
+    const tokenId = match.params.tokenId;
+    const [ nft, setNft ] = useState();
+    const [ ownerId, setOwnerId ] = useState();
+    const [transferNftDetail, setTransferNftDetail] = useState();
+
+    useEffect(() => {
+        NonFungibleTokens.getMetadata(contractId).then(contractMetadata => {
+            NonFungibleTokens.getToken(contractId, tokenId, contractMetadata.base_uri).then(token => {
+                token.contract_id = contractId;
+                setNft(token);
+                setOwnerId(token.owner_id);
+            });
+        });
+    }, []);
+
+    return (
+        <StyledContainer className='medium centered'>
+          {
+            nft && 
+            <div className='container'>
+                <BackArrowButton
+                    onClick={() => history.goBack()}
+                    className='back-btn'
+                >
+                </BackArrowButton>
+
+                <img src={nft.metadata.mediaUrl} alt='NFT'/>
+                <h1 className="title">{nft.metadata.title}</h1>
+                <p className="desc">{nft.metadata.description}</p>
+
+                <div className='owner'>
+                    <p><Translate id='NFTDetail.owner'/></p>
+
+                    <div className='inner'>
+                        <UserIcon>
+                            <UserIconGrey color='#9a9a9a' />
+                        </UserIcon>
+                        <span>
+                            { ownerId }
+                        </span>
+                    </div>
+                </div>
+
+                {(ownerId === accountId) &&
+                  <FormButton 
+                    className='transfer-btn'
+                    color='gray-gray' 
+                    disabled={ownerId !== accountId || !nearBalance}
+                    onClick={() => setTransferNftDetail(nft)}
+                >
+                    {transferSVG()}
```

Comment:
Let's re-use the [`<SendIcon/>`](https://github.com/near/near-wallet/blob/master/packages/frontend/src/components/wallet/Wallet.js#L383) here instead :)